frontend/src/views/main/project/TestSuiteNewConfiguration.vue

```diff
@@ -0,0 +1,3 @@
+<template>
```

Comment:
not used

Reply:
Actually it's used for the tab `Configuration` which is not used yet. Should we remove the tab altogether?

Reply:
ah, no, sorry, let's leave it then